doc/content/lorawan/lora-phy-format/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "LoRa Physical Layer Packet Format"
+section: Additional Information
+description: ""
+weight:
+---
+
+LoRa uses two types of packet formats for data transmission: explicit and implicit. 
+
+In **explicit** mode, a LoRa packet includes the following elements. 
+
+- **Preamble** is used to synchronize the receiver with the transmitter. It MUST consist of 8 symbols for all regions as mentioned in the LoRaWAN Regional Parameters document. However, the radio transmitter will add another 4.25 symbols resulting in a final preamble length of 8 + 4.25 = 12.25 symbols.
+
+- **PHDR (Physical Header)** is an optional element only present in the explicit mode that contains information about,
+Payload size - indicates the size of the payload.
+Cyclic Redundancy Check (CRC) - indicates the presence or absence of the Payload CRC.
```

Comment:
```suggestion
- **PHDR (Physical Header)** is an optional element only present in the explicit mode that contains information about payload size and CRC (Cyclic Redundancy Check).
```

Reply:
If you use PHDR (Physical Header), please be consistent and use CRC (Cyclic Redundancy Check) and not Cyclic Redundancy Check (CRC). This improves readability.